test/mocaccino-test.js

```diff
@@ -54,11 +127,11 @@ function passOutputAssert(done) {
     if (err) {
       done(err);
     } else {
-      assert.equal(out, '1..1\n'
-        + 'ok 1 fixture passes\n'
+      assert.equal(out, 'ok 1 fixture passes\n'
         + '# tests 1\n'
         + '# pass 1\n'
-        + '# fail 0\n');
+        + '# fail 0\n'
+        + '1..1\n');
```

Comment:
I don't know too much about the TAP spec, but it seems the plan seems to get printed after the results now? I think it's this commit: https://github.com/mochajs/mocha/commit/2078c3203f55ee0e1783dc2fc9e20ac51dcb6896#diff-e8a8c7ad26fed3d12b4bd90577d2fb603f34e78ac146e603c043964f96f5a37e that is causing it. I have no idea if this will be a problem when using this in Mochify?

Reply:
When linking this version of mocaccino into my local version of mochify this will also make quite a few tests fail because they assert the previous format.

Reply:
Seems TAP is fine with the plan being either printed at the beginning or the end of the output https://testanything.org/tap-specification.html#the-plan so this is not a bug or anything.

Reply:
Thank you for researching. This will be a major release, and also a major of mochify, so it's fine if the output format changes.